src/fusion.cpp

```diff
@@ -108,6 +108,50 @@ miopenStatus_t ConvBiasActivFusion(Handle& handle,
     return miopenStatusSuccess;
 }
 
+static auto MakeConvBiasActivFusionInvokeParams(const FusionContext& context,
+                                                const FusionDescription& problem,
+                                                miopen::OperatorArgs& params)
+{
+    MIOPEN_LOG_I2("Calling function MakeConvBiasActivFusionInvokeParams");
```

Comment:
All the MIOPEN_LOG_* macros print the name of the function. Add to it something useful or nothing. I highly recommend printing the buffer addresses and sizes.

Please also change the level from Info2 to Info.